pkg/controller/revision/revision_controller.go

```diff
@@ -325,6 +336,45 @@ func (r *ReconcileRevision) deleteRevision(log logr.Logger, revision *picchuv1al
 	return nil
 }
 
+func (r *ReconcileRevision) CreateSentryReleaseForRevision(log logr.Logger, revision *picchuv1alpha1.Revision, config utils.Config) (sentry.Release, error) {
+	tag, foundtag := revision.Labels[picchuv1alpha1.LabelTag]
+	ref, foundref := revision.Labels[picchuv1alpha1.LabelCommit]
+	repo, foundrepo := revision.Annotations[picchuv1alpha1.AnnotationRepo]
+	app, foundapp := revision.Labels[picchuv1alpha1.LabelApp]
+
+	if foundtag && foundref && foundrepo && foundapp {
+		log.Info("Registering release with Sentry", "Name", revision.Name, "Namespace", revision.Namespace, "Version", tag, "Ref", ref)
+		client, _ := sentry.NewClient(config.SentryAuthToken, nil, nil)
```

Comment:
Any reason why you ignore the error? Have you done any eyeballing of how long client creation/initialization takes? Is it worth sharing a global client as part of the reconciler config object that gets init'd on startup, with a possible panic() if the client doesn't init?

Reply:
Looks like the only error is empty string endpoint, but maybe that changes in the future?